src/config/backpack/crud.php

```diff
@@ -466,5 +466,15 @@
             'crud::filters', // falls back to 'resources/views/vendor/backpack/crud/filters'
         ],
     ],
+    // the uploaders for the `withFiles` macro
+    'uploaders' => [
+        'withFiles' => [
+            'image'           => \Backpack\CRUD\app\Library\Uploaders\SingleBase64Image::class,
+            'upload'          => \Backpack\CRUD\app\Library\Uploaders\SingleFile::class,
+            'upload_multiple' => \Backpack\CRUD\app\Library\Uploaders\MultipleFiles::class,
+        ],
+    ],
+
+    'file_name_generator' => \Backpack\CRUD\app\Library\Uploaders\Support\FileNameGenerator::class,
```

Comment:
Let's merge filename into `uploaders` like we talked:

```suggestion

    'uploaders' => [
        'macros' = [
            'withFiles' => [
                'image'           => \Backpack\CRUD\app\Library\Uploaders\SingleBase64Image::class,
                'upload'          => \Backpack\CRUD\app\Library\Uploaders\SingleFile::class,
                'upload_multiple' => \Backpack\CRUD\app\Library\Uploaders\MultipleFiles::class,
            ],
        ],

        'filename_generator' => \Backpack\CRUD\app\Library\Uploaders\Support\FileNameGenerator::class,
    ],
```